src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -98,18 +98,35 @@ export default Vue.extend({
     },
 
     invidiousUrl: function () {
-      return `${this.currentInvidiousInstance}/watch?v=${this.id}`
+      let videoUrl = `${this.currentInvidiousInstance}/watch?v=${this.id}`
+      // `playlistId` can be undefined
+      if (this.playlistId && this.playlistId.length !== 0) {
+        // `index` seems can be ignored
+        videoUrl += `&list=${this.playlistId}`
+      }
+      return videoUrl
     },
 
     invidiousChannelUrl: function () {
       return `${this.currentInvidiousInstance}/channel/${this.channelId}`
     },
 
     youtubeUrl: function () {
-      return `https://www.youtube.com/watch?v=${this.id}`
+      let videoUrl = `https://www.youtube.com/watch?v=${this.id}`
+      // `playlistId` can be undefined
+      if (this.playlistId && this.playlistId.length !== 0) {
+        // `index` seems can be ignored
+        videoUrl += `&list=${this.playlistId}`
+      }
+      return videoUrl
     },
 
     youtubeShareUrl: function () {
+      // `playlistId` can be undefined
+      if (this.playlistId && this.playlistId.length !== 0) {
+        // `index` seems can be ignored
+        return `https://www.youtube.com/watch?v=${this.id}&list=${this.playlistId}`
```

Comment:
The shortened, https://youtu.be does also support playlists:
`https://youtu.be/watch?v=${this.id}&list=${this.playlistId}`
e.g.
`https://youtu.be/watch?v=NWsMzSe5gLw&list=PL8mG-RkN2uTyfj8UNijuKK6Y38Cz7eQKJ`

Whether the slightly shorter url matters is up to you; but at least then you would get the same domain back from this function regardless of playlist or not.

Reply:
Thanks for letting me know
Will update soon

I was just copying this from an existing implementation

Reply:
I found out that
`https://youtu.be/${this.id}?list=${this.playlistId}` also works
`https://youtu.be/bStmUUGMA0A?list=PLt6FSN4PoPuSzKNYTjGpv133iaK_i2JyZ`
Gonna update to that